### ISSUE TITLE
Adhesion of gallbladder

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10052]
**Preferred term label:**
Adhesion of gallbladder

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0267902

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Gallstone ileus

**Your nano-attribution (ORCID)**
